jekyll/_config.yml

```diff
@@ -74,7 +74,6 @@ liquid:
 
 asciidoc:
   attributes:
-      source-highlighter: rouge
```

Comment:
@rosieyohannan I need to remove this to make PrismJS work in `.adoc` files but I am wondering if there is some sort of impact on the generated pdf?

I tried to generate pdf locally with `./scripts/build_pdfs_asciidoc.sh` and I am still seeing highlighted code: 
<img width="768" alt="Screen Shot 2022-02-04 at 6 05 39 PM" src="https://user-images.githubusercontent.com/1449325/152624816-54356767-8114-4242-bf5f-329a1cf97878.png">

Am I in the clear?

Reply:
Yes that looks fine! I guess we didn't need that attribute :-D